operations/research-and-development/product/release-process/boards-release.md

```diff
@@ -1,35 +1,34 @@
-# Focalboard Release Process
+# Boards Release Process
 
-This document outlines the Focalboard release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
+This document outlines the Boards release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
 
 ## Release Timeline
 
-Focalboard release schedule for July and August is outlined here: [https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl\_oJ-bAz1\_qpEiGhgF4HdDX1Uf4Q/edit\#gid=0](https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl_oJ-bAz1_qpEiGhgF4HdDX1Uf4Q/edit#gid=0).
+Boards release schedule is outlined here: 
```

Comment:
Let me know if this is outlined somewhere.

Reply:
The Boards monthly release process is currently captured in the [Monthly Release Playbook](https://community-daily.mattermost.com/playbooks/playbooks/xz7uzn4ui7fpice9xn3dyaeu6h/preview).

As for the key dates each month, @wuwinson will own those going forward, and they will align with the Mattermost dates [per this thread](https://community-daily.mattermost.com/core/pl/dx7e8716sig75gu1qzj7rsarzr).